tests/sparsity/test_common.py

```diff
@@ -44,6 +44,7 @@ def test_can_create_rb_algo__with_adaptive_scheduler():
     assert isinstance(compression_ctrl.scheduler, AdaptiveSparsityScheduler)
 
 
+@pytest.mark.skip()
```

Comment:
AR @andrey-churkin to either restore the test with new premises or remove it altogether in a next PR